holoviews/plotting/mpl/raster.py

```diff
@@ -183,7 +183,7 @@ def update_handles(self, key, axis, element, ranges, style):
 class ImagePlot(RasterPlot):
 
     def get_data(self, element, ranges, style):
-        data = element.dimension_values(2, flat=False)
+        data = np.flipud(element.dimension_values(2, flat=False))
```

Comment:
How come this doesn't flip ``Image`` given ``dimension_values`` for numpy arrays won't be based on interface classes?